srcpkgs/lsof/patches/fix-tests.patch

```diff
@@ -0,0 +1,14 @@
+--- a/tests/LTsock.c
++++ b/tests/LTsock.c
+@@ -255,10 +255,7 @@ main(argc, argv)
+  * Get the host name and its IP address.  Convert the IP address to dotted
+  * ASCII form.
+  */
+-    if (gethostname(hnm, sizeof(hnm) - 1)) {
+-	cem = "ERROR!!!  can't get this host's name";
+-	goto print_errno;
+-    }
++    strncpy(hnm, "localhost", sizeof(hnm) - 1);
```

Comment:
Add a simple comment on top of the patch saying this is done so a `hosts` file isn't necessary.

Reply:
done.